### PR TITLE
Rename rpc.proto to imrpc.proto

### PR DESCRIPTION
**Pull request by derekbit**
To avoid the "protobuf namespace conflicts" error, rename the protocol buffer file.

Depends on https://github.com/longhorn/longhorn-instance-manager/pull/108

https://github.com/longhorn/longhorn/issues/3796

Signed-off-by: Derek Su <derek.su@suse.com>

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -75,3 +75,5 @@ require (
 	sigs.k8s.io/controller-runtime v0.4.0
 	sigs.k8s.io/yaml v1.2.0
 )
+
+replace github.com/longhorn/longhorn-instance-manager v0.0.0-20210729081215-50c310f97378 => github.com/derekbit/longhorn-instance-manager v0.0.0-20220411065951-c052142954b7
```
Comment (innobead) [inline go.mod:79]: Why using replace instead of updating the required dep directly?

**Review by innobead — APPROVED** [review]
LGTM

After updating this, if running `go mod tidy` then `go test or make test`, all tests passed w/o any proto registration error?

File: go.mod (hunk 1 above)
Comment (innobead) [reply go.mod:79]: Oh, we are still waiting for https://github.com/longhorn/longhorn-instance-manager/pull/108 😄

**Comment by derekbit** [comment]
> LGTM
> 
> After updating this, if running `go mod tidy` then `go test or make test`, all tests passed w/o proto registration error anymore, right?

yeah, works as expected while doing
```
go mod tidy
go mod vendor
make or make test
```

**Review by innobead — APPROVED** [review]

**Pull request merged**